README.md

```diff
@@ -1,14 +1,15 @@
 # Exercism Elixir Track
 
-![build status](https://travis-ci.org/exercism/elixir.svg?branch=master)
+![GitHub branch checks state](https://img.shields.io/github/checks-status/exercism/elixir/main)
```

Comment:
I got this from https://shields.io/category/build but I'm not sure if it works... it says "checks: pending" for me. But the old one definitely doesn't work anymore 😬

Reply:
If you go into the repo actions tab, select a workflow, then click the `...` there is a badge button there, where a modal will pop up with that badge snippet, like this one for configlet

```
[![configlet](https://github.com/exercism/elixir/actions/workflows/configlet.yml/badge.svg)](https://github.com/exercism/elixir/actions/workflows/configlet.yml)
```

Reply:
Right, but that forces me to choose a workflow. I'm looking for a badge that would show the "checks" status, which includes adding together the statuses of many workflows. Basically a badge equivalent of this green tick :)

![Screen Shot 2021-06-14 at 10 17 43](https://user-images.githubusercontent.com/7852553/121861190-e7094700-ccf9-11eb-8620-7f6d0584d47f.png)

Reply:
Sounds good, let's try it then 🤷‍♂️👍

Reply:
Looks like "just merging" didn't fix the problem 😬 I will have to debug it later